packages/babel-plugin-strip-runtime/src/__tests__/strip-runtime-source-code.test.ts

```diff
@@ -96,8 +96,8 @@ describe('babel-plugin-strip-runtime using source code', () => {
         expect(actual).toMatchInlineSnapshot(`
           "/* app.tsx generated by @compiled/babel-plugin v0.0.0 */
 
-          import * as React from 'react';
           import { ax, ix } from '@compiled/react/runtime';
+          import * as React from 'react';
```

Comment:
any reason these are no longer top imports?

Reply:
Oh, I can fix that. Sec.